packages/service-configuration/service-configuration.d.ts

```diff
@@ -5,6 +5,12 @@ export interface Configuration {
   secret: string;
 }
 
+class ConfigError extends Error {
+  constructor(serviceName?: string);
+  message: string;
+}
+
 export declare var ServiceConfiguration: {
   configurations: Mongo.Collection<Configuration>;
+  ConfigError
```

Comment:
Sorry for being late, but this is incorrect: with this syntax, `ConfigError` is `any`. It should say `ConfigError: typeof ConfigError` (without the `typeof` it'd be an instance and not the class itself).